specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2022-11-01-preview/hybridcloud.json

```diff
@@ -0,0 +1,992 @@
+          "items": {
+            "$ref": "#/definitions/CloudConnection"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "A list of Cloud Connection resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to hold the next page of results"
+        }
+      },
+      "description": "Response for the ListCloudConnections API service call"
+    },
+    "CloudConnectionPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "cloudConnectorId": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The Cloud Connector to which the Cloud Connection belongs."
+        },
+        "virtualHub": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The virtualHub to which the Cloud Connection belongs."
+        },
+        "remoteResourceId": {
+          "type": "string",
+          "description": "Identifier for the remote resource"
+        },
+        "sharedKey": {
+          "type": "string",
+          "description": "Shared Key of the VPN Link Connection"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of the application rule collection resource."
+        }
+      },
+      "description": "Cloud Connector resource properties."
+    },
+    "CloudConnection": {
+      "type": "object",
+      "description": "Resource which represents the managed network connection between Azure Gateways and remote cloud gateways",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudConnectionPropertiesFormat",
+          "description": "Properties of the Cloud Connection."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "DiscoverResources": {
+      "type": "object",
+      "description": "Resource that represents resources found in remote cloud environments",
+      "properties": {
+        "resourceFilter": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ResourceFilter",
+          "description": "Remote Cloud Resource Type filter"
+        },
+        "discoveredResources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DiscoveredResource"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "A list of discovered remote cloud resources."
+        }
+      }
+    },
+    "ResourceFilter": {
+      "type": "object",
+      "description": "Remote Cloud Resource Type filter",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Resource name.",
+          "readOnly": true
```

Comment:
>readOnly [](http://example.com/codeflow?start=11&length=8)

This shouldn't be readonly if its a query parameter in a filter I would think?? Maybe I misunderstand the scenario?

Reply:
And why are there no filter properties??

Reply:
Yes, this is a filter. should not be readonly